### DISCUSSION TITLE
second link is no longer accessible

### DISCUSSION THREAD

**Discussion by Tigger86** [arkypita/Vigotech-VG-L7X#11] [Q&A]
Hello,
I try to follow the instructions, unfortunately I lack the expertise to find an alternative for the second link. 
unfortunately the link is no longer current

[https://github.com/arkypita/Vigotech-VG-L7X/tree/main/CustomFirmware-VG-L7x](url)

**Comment by arkypita** [comment]
Where did you find this link? so I can correct it

Right one: https://github.com/arkypita/Vigotech-VG-L7X/tree/main/CustomFirmware-VG-L7x

**Comment by Tigger86** [comment]
Hey, thanks for getting back to me so quickly.
So the link is directly with the instructions for flashing.Here the link. I get an error 404https://github.com/espressif/arduino-esp32/blob/master/docs/arduino-ide/boards_manager.mdAnd here a Screenshot about what i mean:Hope u understand what i mean.Thanks 

[Screenshot](https://ibb.co/8YfqXHJ)


Am 25.07.2021 19:08 schrieb arkypita ***@***.***>:
Where did you find this link? so I can correct it
Right one: https://github.com/arkypita/Vigotech-VG-L7X/tree/main/CustomFirmware-VG-L7x

**Reply by Tigger86** [reply]
Is there no alternative?

**Comment by Brainbug01** [comment]
![image](https://user-images.githubusercontent.com/35106961/148770648-d4fd7fb8-abac-4e20-8ad9-b7e82592e590.png)

https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_index.json

![image](https://user-images.githubusercontent.com/35106961/148770336-ad5d4709-ed1b-4110-a187-89e9d2c229b0.png)
Version 1.0.5

![image](https://user-images.githubusercontent.com/35106961/148770516-ebb282a8-2d70-46dd-a596-e9172a349924.png)
WifiManager

before the cable was connected to the PC, I had to start boot mode
